packages/nextjs/package.json

```diff
@@ -78,12 +78,11 @@
     "@types/react": "*",
     "@types/react-dom": "*",
     "eslint-config-custom": "*",
-    "next": "^13.4.19",
```

Comment:
Same as with expo, we'll need to keep this as a devDep

Reply:
I think we can remove this from here if we have it as a `devDependency` in the root `package.json` but what's the benefit in doing that?

Reply:
i will keep it as dev dependency here.